contexts/webkms-v1.jsonld

```diff
@@ -0,0 +1,24 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "@protected": true,
+
+    "AesKeyWrappingKey2019": "https://w3id.org/security#AesKeyWrappingKey2019",
+    "DeleteKeyOperation": "https://w3id.org/security#DeleteKeyOperation",
+    "DeriveSecretOperation": "https://w3id.org/security#DeriveSecretOperation",
+    "ExportKeyOperation": "https://w3id.org/security#ExportKeyOperation",
+    "GenerateKeyOperation": "https://w3id.org/security#GenerateKeyOperation",
+    "KmsOperation": "https://w3id.org/security#KmsOperation",
+    "RevokeKeyOperation": "https://w3id.org/security#RevokeKeyOperation",
+    "Sha256HmacKey2019": "https://w3id.org/security#Sha256HmacKey2019",
+    "SignOperation": "https://w3id.org/security#SignOperation",
+    "UnwrapKeyOperation": "https://w3id.org/security#UnwrapKeyOperation",
+    "VerifyOperation": "https://w3id.org/security#VerifyOperation",
+    "WrapKeyOperation": "https://w3id.org/security#WrapKeyOperation",
+    "kmsModule": "https://w3id.org/security#kmsModule",
+    "unwrappedKey": "https://w3id.org/security#unwrappedKey",
+    "verifyData": "https://w3id.org/security#verifyData",
+    "wrappedKey": "https://w3id.org/security#wrappedKey"
+  }
```

Comment:
See webkms-client operations, e.g., https://github.com/digitalbazaar/webkms-client/blob/main/KmsClient.js#L63-L67 for additional required terms.

Reply:
Also useful: https://github.com/digitalbazaar/webkms-switch/tree/main/lib/schemas

Reply:
@dlongley I double-checked against both those links, everything from them appears here in this context. (I'm still not fully clear whether `verifyData` belongs here or in the zCap context.)

Reply:
@dmitrizagidulin -- there's a term missing from what's highlighted in the first link: `invocationTarget`. Please double check the other operations and their results to make sure nothing else is missing.

Reply:
Or is your plan to make it so that KMS operations must include both the webkms context and the zcap context?

Reply:
Where is `verifyData` used by zcaps? I would think it belongs here -- it appears in KMS operations.

Reply:
@dlongley yeah, `invocationTarget` is currently living with other zCap related terms, over in https://github.com/digitalbazaar/zcap-context/pull/1/files#diff-556abca0469ecdd34d72738d6e7ba035f9350e7b17416347d66420d821eba5faR16

I think we can't avoid needing both a KMS and a zCap context, on operations? Or can we?

Reply:
(Added `invocationTarget` and `signatureValue`; let's see if we can get away with just using the KMS context.)